roles/kube_cert_manager/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+debug: False
+
+cert_manager_version: '0.2.4'
+cert_manager_namespace: 'kube-ops'
```

Comment:
Maybe `kube-ingress` is more appropriate?

Reply:
yes, indeed...